services/reader.go

```diff
@@ -88,9 +123,18 @@ func ReadColumns(file multipart.File, sheetsToExtract []string) (Output, error)
 				return nil, err
 			}
 
+			if currentColIndex < defaultColIndex {
+				currentColIndex++
+				continue
+			}
+
 			rowValue := make([]interface{}, 0)
 
-			for _, cellValue := range col {
+			for index, cellValue := range col {
```

Comment:
It would be better to use a classic loop and start the iteration to the correct row instead passing X rows before the first one.

For example, if the `defaultRowIndex` is 45, you will iterate 44 times for nothing